server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -363,6 +396,9 @@ public Tablet(final TabletServer tabletServer, final KeyExtent extent,
       });
     }
 
+    storagePolicy = tableConfiguration.newDeriver(conf -> conf.get(Property.TABLE_STORAGE_POLICY));
```

Comment:
Could use the Policies type here.  Have a single deriver that returns a Policies object.

Reply:
ok